includes/admin/dashboard-widgets.php

```diff
@@ -170,7 +170,7 @@ function edd_load_dashboard_sales_widget( ) {
 					<a href="<?php echo esc_url( $link ); ?>">
 						<?php
 						$customer      = edd_get_customer( $payment->customer_id );
-						$customer_name = $customer->name ? $customer->name : __( 'No Name', 'easy-digital-downloads' );
+						$customer_name = isset( $customer->name ) ? $customer->name : __( 'No Name', 'easy-digital-downloads' );
```

Comment:
@robincornett Maybe fallback to email here?

Reply:
I ended up adding the `isset` check because I managed to create/add an order which is apparently not associated with any customer at all, so `edd_get_customer()` does not return an object. Adding the check brings this more in line with a similar check in `class-payments-table.php`, which is where I originally found the "No Name" fallback.

Reply:
Good call on the consistency. I'm not sure it stands out enough as _missing data_ in either place, but keeping them consistent is fine for now, and we can iterate both areas later if needed.